<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1323</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1321</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>281</width>
        <height>671</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
	font: 900 9pt &quot;Segoe UI&quot;;
	font: 900 9pt &quot;Segoe UI&quot;;
	font: 16pt &quot;Segoe UI&quot;;
  
	background-color: rgb(135, 206, 235);
    border: 2px solid #4682B4;
    border-radius: 8px;
    margin-top: 10px;
}
QPushButton {
    background-color: #4682B4;
    color: white;
    border-radius: 5px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #5A9BD5;
}</string>
      </property>
      <property name="title">
       <string>‚ûïAjouter employ√©:</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>üë®‚ÄçüíªID Employ√©:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>üî†Nom:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>üî†Pr√©nom:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>üíºPoste:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>310</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>üî£Mot de passe:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>390</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>440</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>üìûNum t√©l√©phonique:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>480</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>530</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>üìÜDate d'embauche:</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>570</y>
         <width>110</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>620</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>480</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>+216</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>180</y>
        <width>991</width>
        <height>421</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>991</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>991</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #00008B;   /* ÿ£ÿ≤ÿ±ŸÇ ÿ∫ÿßŸÖŸÇ */
    color: white;                /* ÿßŸÑŸÜÿµ ÿ£ÿ®Ÿäÿ∂ */
    gridline-color: white;       /* ŸÑŸàŸÜ ÿßŸÑÿÆÿ∑Ÿàÿ∑ */
    border-radius: 8px;          /* ÿ≠ŸàÿßŸÅ ŸÖÿ≥ÿ™ÿØŸäÿ±ÿ© */
}
QHeaderView::section {
    background-color: #34495e;   /* ÿ±ŸÖÿßÿØŸä ÿ∫ÿßŸÖŸÇ ŸÑŸÑŸÄ header */
    color: white;
    padding: 4px;
    border: 1px solid #2c3e50;
</string>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>141</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prenom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Poste</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mot de passe</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Num t√©l√©phonique:</string>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date d'embauche:</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>30</y>
        <width>411</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:700; color:#00008b;&quot;&gt;Gestion employ√©s&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>120</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(70, 130, 180);
font: 14pt &quot;Segoe UI&quot;;
font: 900 12pt &quot;Segoe UI&quot;;

    color: white;                /* ŸÑŸàŸÜ ÿßŸÑŸÜÿµ */
    border-radius: 12px;         /* ÿßŸÑÿ≠ŸàÿßŸÅ ŸÖÿØŸàŸëÿ±ÿ© */
    padding: 8px 16px;</string>
      </property>
      <property name="text">
       <string>Afficher üîç</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_13">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>130</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>130</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Afficher employ√©:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>610</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(70, 130, 180);
font: 14pt &quot;Segoe UI&quot;;
font: 900 12pt &quot;Segoe UI&quot;;

    color: white;                /* ŸÑŸàŸÜ ÿßŸÑŸÜÿµ */
    border-radius: 12px;         /* ÿßŸÑÿ≠ŸàÿßŸÅ ŸÖÿØŸàŸëÿ±ÿ© */
    padding: 8px 16px;</string>
      </property>
      <property name="text">
       <string>Supprimer üóë</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_14">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>620</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>619</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Supprimer employ√©:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>130</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Trier par:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>130</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Poste </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>610</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(70, 130, 180);
font: 14pt &quot;Segoe UI&quot;;
font: 900 12pt &quot;Segoe UI&quot;;

    color: white;                /* ŸÑŸàŸÜ ÿßŸÑŸÜÿµ */
    border-radius: 12px;         /* ÿßŸÑÿ≠ŸàÿßŸÅ ŸÖÿØŸàŸëÿ±ÿ© */
    padding: 8px 16px;</string>
      </property>
      <property name="text">
       <string>Exporterüìë</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Statistiques</string>
     </attribute>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>90</y>
        <width>751</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Capture d‚Äô√©cran 2025-09-29 142122.png</pixmap>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1323</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
